content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -17,6 +17,8 @@ Terraform Enterprise via the Installer. Customers using the AMI can follow the
 
 ## Migrating from AMI
 
+!> **Deprecation warning**: The AMI will no longer be actively developed as of 201808-1 and will be fully decommissioned on November 30, 2018. Please see our [Migration Guide](./migrate.html) for instructions to migrate to the new Private Terraform Enterprise Installer.
```

Comment:
It looks like this page already has a link to the migration guide, directly below, so let's leave out the second sentence in this case.

Reply:
👍